Gruntfile.js

```diff
@@ -137,7 +137,7 @@ module.exports = function( grunt ) {
 						"ascii_only": true
 					},
 					banner: "/*! jQuery v<%= pkg.version %> | " +
-						"(c) 2005, <%= grunt.template.today('yyyy') %> jQuery Foundation, Inc. | " +
+						"Copyright jQuery Foundation and other contributors " +
```

Comment:
I'd leave `|` at the end, it separates the copyright info from the URL to the license (see the linked minified file for example).